dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/relationship/RelationshipValidator.java

```diff
@@ -53,7 +53,8 @@ public RelationshipValidator() {
                 new MetaValidator(),
                 new LinkValidator(),
                 new ConstraintValidator(),
-                new DuplicationValidator()));
+                new DuplicationValidator(),
+                securityOwnershipValidator));
```

Comment:
I would put this new validator right after the `MetaValidator`.
As soon as it is validated that the relationship type exists we should check if the user has access to it.
Running this validator as the last one could potentially disclose some information that the user shouldn't see, for example, the user shouldn't be informed that there is a duplicated relationship if it has no access to the relationship type.